src/pages/components/InputFieldEnergy.js

```diff
@@ -11,7 +11,7 @@ export default function InputFieldEnergy({ val, onChangeVal, name }) {
         name={name}
         value={val}
         onChange={(e) => onChangeVal(e)}
-        placeholder="0.0 kWh"
+        placeholder="0.0 MWh"
```

Comment:
Can I check where is this value being transformed into `Wh` for the api post? I can't find it happening anywhere

Reply:
In `CertificateViewDetails` line `76` and `84` e.g: `{energy / 1000000}`